larpandoracontent/LArTrackShowerId/MvaPfoCharacterisationAlgorithm.h

```diff
@@ -37,12 +37,18 @@ class MvaPfoCharacterisationAlgorithm : public PfoCharacterisationBaseAlgorithm
     pandora::StatusCode ReadSettings(const pandora::TiXmlHandle xmlHandle);
 
     ClusterCharacterisationFeatureTool::FeatureToolVector m_featureToolVector;         ///< The feature tool map
-    PfoCharacterisationFeatureTool::FeatureToolVector m_featureToolVectorThreeD;       ///< The feature tool map for 3D info
-    PfoCharacterisationFeatureTool::FeatureToolVector m_featureToolVectorNoChargeInfo; ///< The feature tool map for missing W view
+
+    PfoCharacterisationFeatureTool::FeatureToolMap m_featureToolMapThreeD;             ///< FeatureToolMap as a map for 3D info
+    PfoCharacterisationFeatureTool::FeatureToolMap m_featureToolMapNoChargeInfo;       ///< FeatureToolMap as a map for missing W view
+
+    std::vector<std::string> m_algorithmToolNames;                                     ///< Vector of strings saving feature tool order for use in feature calculation
+    std::vector<std::string> m_algorithmToolNamesNoChargeInfo;                         ///< Vector of strings saving feature tool order for use in feature calculation (missing W view)
```

Comment:
pandora::StringVector

Reply:
I think that's fixed by my most recent commit, no?

Reply:
It doesn't look like it's made it into the pull request.

Reply:
Maybe it took a bit to update or something? I see it when I go to the FilesChanged tab

Reply:
That's odd, I still see

`std::vector<std::string> m_algorithmToolNames;
std::vector<std::string> m_algorithmToolNamesNoChargeInfo; `

rather than

`pandora::StringVector ...`

Reply:
This is what I see 🤔 -- hopefully the pasted image works in GitHub...

<img width="1053" alt="image" src="https://user-images.githubusercontent.com/61521590/181781870-1fce652f-33ce-4537-8c5d-c13722684d0c.png">

Reply:
github still shows me the unmodified version, but if I navigate to the branch that triggered the PR I can see the update, so presumably it will merge without issue.